src/content/en/fundamentals/_toc-base.yaml

```
@@ -13,6 +13,7 @@ toc:
       path: /web/fundamentals/primers/modules
     - title: Promises
       path: /web/fundamentals/primers/promises
```

```suggestion
      path: https://web.dev/SOMETHING
```

Set the URL to the actual URL it will go to, that way you eliminate the redirect.